### PR TITLE
chore/refactoring and PR cleanup

### PR DESCRIPTION
**Pull request by m24murray**
- Refactoring and fixing failing tests as per PR comments from https://github.com/Civil-Service-Human-Resources/oauth2-stub/pull/23

### CODE REVIEW

File: src/main/java/uk/gov/cshr/service/InviteService.java [hunk 1]
```diff
@@ -14,14 +17,15 @@
...
 import java.util.HashMap;
 
+@Service
+@Transactional
 public class InviteService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(InviteService.class);
     private static final String EMAIL_PERMISSION = "email";
     private static final String ACTIVATION_URL_PERMISSION = "activationUrl";
     private static final long EXPIRATION_IN_MILLISECONDS = 86400000; // 24hrs = 86400000ms
```
Comment (elxsy) [inline src/main/java/uk/gov/cshr/service/InviteService.java]: CHORE - This should come from configuration

File: src/main/java/uk/gov/cshr/service/InviteService.java [hunk 2]
```diff
@@ -60,6 +71,7 @@ public void sendEmail(Invite invite) throws NotificationClientException {
         NotificationClient client = new NotificationClient(api);
         SendEmailResponse response = client.sendEmail(templateId, invite.getForEmail(), personalisation, "");
 
+        // TODO: 25/04/2018 Matt - remove this log later, just using for dev purposes to output email contents so we can get the signup activation link
```
Comment (elxsy) [inline src/main/java/uk/gov/cshr/service/InviteService.java]: DOD - Left debug code

File: src/main/java/uk/gov/cshr/service/InviteService.java (hunk 1 above)
Comment (m24murray) [reply src/main/java/uk/gov/cshr/service/InviteService.java]: Just to clarify, do you just mean the EXPIRATION_IN_MILLISECONDS, or all 3 fields?

File: src/main/java/uk/gov/cshr/service/InviteService.java (hunk 2 above)
Comment (m24murray) [reply src/main/java/uk/gov/cshr/service/InviteService.java]: This was intentional for now as we are still using the test api key. When we cut over, ill take it out. But until then I don't want to forget about it

Comment (elxsy) [reply src/main/java/uk/gov/cshr/service/InviteService.java]: OK

File: src/main/java/uk/gov/cshr/service/InviteService.java (hunk 1 above)
Comment (elxsy) [reply src/main/java/uk/gov/cshr/service/InviteService.java]: just EXPIRATION_IN_MILLISECONDS as comment for the line.

**Pull request merged**